.gitlab/functional_test/system_probe.yml

```diff
@@ -5,6 +5,31 @@
 # include:
 #   - /.gitlab/kitchen_common/testing.yml
 
+.pull_test_dockers:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
```

Comment:
You are using the `x64` image for both architectures, is that intentional?

Reply:
yes, or more exactly I'm agnostic to that
in `save-test-dockers` task, I get the wanted architecture as a parameter and download the relevant dockers by
```python3
ctx.run(f"docker pull --platform linux/{arch} {image}")
```